lib/stencil-bundle.js

```diff
@@ -292,6 +302,11 @@ function bundleTaskRunner(callback) {
         if (err) {
             return callback(err);
         }
+        const distExists = this.distFolderCheck();
+
+        if (!distExists) {
```

Comment:
What about `options.dest`?

Reply:
this.options.dest is undefined for me.

Reply:
I just realized this.options.dest is an option for outputFolder which this.themePath can also substitute for.  For distExists to be true, this.themePath has to exist.  I can be more explicit about that though if you would like.

Reply:
If `options.dest` is provided, then the check in `distFolderCheck` will not work because it assumes `this.themePath`.